systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaTopicUtils.java

```diff
@@ -219,4 +223,31 @@ public static void waitForTopicWithPrefixDeletion(String namespaceName, String t
                 }
             });
     }
+
+    /**
+     * Verifies that {@code absentTopicName} topic remains absent in {@code clusterName} Kafka cluster residing in {@code namespaceName},
+     * by querying the cluster using kafka scripts from {@code queryingPodName} Pod.
+     * @param namespaceName Namespace name
+     * @param queryingPodName  the name of the pod to query Kafka topics from
+     * @param clusterName name of Kafka cluster
+     * @param absentTopicName name of Kafka topic which should not be created.
+     * @throws AssertionError in case topic is created
+     */
+    public static void verifyUnchangedTopicAbsence(String namespaceName, String queryingPodName, String clusterName, String absentTopicName) {
+
+        long poolInterval = Constants.POLL_INTERVAL_FOR_RESOURCE_READINESS; // repeat every second
+        long duration = Constants.RECONCILIATION_INTERVAL; // repeat for ~30sec
+        long startTime = System.currentTimeMillis();
+        long endTime = startTime + duration;
+        LOGGER.info("Verify absence of topic '{}' in listed kafka topics for next {} seconds in Namespace/{}", absentTopicName, duration / 1000, namespaceName);
+
+        while (System.currentTimeMillis() < endTime) {
+            assertThat(KafkaCmdClient.listTopicsUsingPodCli(namespaceName, queryingPodName, KafkaResources.plainBootstrapAddress(clusterName)), not(hasItems(absentTopicName)));
+            try {
+                Thread.sleep(poolInterval);
+            } catch (InterruptedException e) {
+                e.printStackTrace();
+            }
+        }
```

Comment:
You should use `TestUtils.waitFor` method instead of this

Reply:
yeah I think that is a bit different thing that i want to test, 
whereas waitFor implies that I am waiting for something (to occur) in this case I am simply making sure that something really remains the way it is (kafka topic "topic-without-label" topic really is not created and never have been from KafkaTopic CR with same name). So this would 
- either force me to either wait for timeout (waiting for it to exist) 
- or pass immediately (waiting for it to not exist) 
- or i would modify it to stability like check which would be actually pointless as i don't want it only to be stable, instead stable immediately => something like unchanged. 

But I am really open to change this implementation.

Reply:
As you mentioned, we are using it for the stability waits also (f.e. `waitUntilPodStabilityReplicasCount`), but yeah, in that case when you don't want to use it as "stability wait", I guess you can keep it